.eslintrc.cjs

```diff
@@ -13,8 +13,14 @@ module.exports = {
     },
     rules: {
         'prettier/prettier': 'warn',
+        '@typescript-eslint/no-unused-vars': [
+            'error',
+            {
+                argsIgnorePattern: '^_',
+                varsIgnorePattern: '^_',
+                caughtErrorsIgnorePattern: '^_',
+            },
+        ],
```

Comment:
This seems a code convention rule.
Is it shared between all powsybl typescript projects ?

Reply:
The change have been reverted